load/resources/cim-qudt.ttl

```diff
@@ -0,0 +1,143 @@
+    rdfs:label "Norwegian Krone per Megawatt Hour" ;
+    rdfs:label "Norwegian Krone per Megawatt Hour"@en ;
+.
+quantitykind:CostPerEnergy qudt:applicableUnit unit:CCY_NOK-PER-MegaW-HR .
+
+
+unit:CCY_EUR-PER-MegaW-HR
+    a qudt:Unit ;
+    qudt:applicableSystem sou:SI ;
+    qudt:conversionMultiplier 0.000000000277777777777777777777777777778 ;
+    qudt:conversionMultiplierSN 2.777777777777777777777777777777778E-10 ;
+    qudt:hasDimensionVector qkdv:A0E0L-2I0M-1H0T2D0 ;
+    qudt:hasFactorUnit [
+        qudt:exponent -1 ;
+        qudt:hasUnit unit:HR ;
+        ] ;
+    qudt:hasFactorUnit [
+        qudt:exponent -1 ;
+        qudt:hasUnit unit:MegaW ;
+        ] ;
+    qudt:hasFactorUnit [
+        qudt:exponent 1 ;
+        qudt:hasUnit unit:CCY_EUR ;
+        ] ;
+    qudt:hasQuantityKind quantitykind:CostPerEnergy ;
+    qudt:plainTextDescription "Unit for measuring the cost of electricity." ;
+    qudt:symbol "€/(MW·h)" ;
+    rdfs:isDefinedBy <http://qudt.org/vocab/unit> ;
+    rdfs:label "Euro per Megawatt Hour" ;
+    rdfs:label "Euro per Megawatt Hour"@en ;
+.
+quantitykind:CostPerEnergy qudt:applicableUnit unit:CCY_EUR-PER-MegaW-HR .
+
+## exchange rate
+
+unit:CCY_DKK-PER-CCY_EUR
+    a qudt:Unit ;
+    qudt:hasQuantityKind quantitykind:ExchangeRate ;
+    qudt:hasFactorUnit [
+        qudt:hasUnit unit:CCY_DKK;
+        qudt:exponent 1
+        ];
+    qudt:hasFactorUnit [
+        qudt:hasUnit unit:CCY_EUR;
+        qudt:exponent -1
+        ];
+    rdfs:label "Danish Krone per Euro" ;
+    qudt:symbol "DKK/EUR" ;
+    qudt:plainTextDescription "Exchange rate of Danish Krone to Euros" ;
+    rdfs:isDefinedBy <http://qudt.org/vocab/unit> ;
+.
+
+unit:CCY_NOK-PER-CCY_EUR
+    a qudt:Unit ;
+    qudt:hasQuantityKind quantitykind:ExchangeRate ;
+    qudt:hasFactorUnit [
+        qudt:hasUnit unit:CCY_NOK;
+        qudt:exponent 1
+        ];
+    qudt:hasFactorUnit [
+        qudt:hasUnit unit:CCY_EUR;
+        qudt:exponent -1
+        ];
+    rdfs:label "Norwegian Krone per Euro" ;
+    qudt:symbol "NOK/EUR" ;
+    qudt:plainTextDescription "Exchange rate of Norwegian Krone to Euros" ;
+    rdfs:isDefinedBy <http://qudt.org/vocab/unit> ;
+.
+
+unit:CCY_SEK-PER-CCY_EUR
+    a qudt:Unit ;
+    qudt:hasQuantityKind quantitykind:ExchangeRate ;
+    qudt:hasFactorUnit [
+        qudt:hasUnit unit:CCY_SEK;
+        qudt:exponent 1
+        ];
+    qudt:hasFactorUnit [
+        qudt:hasUnit unit:CCY_EUR;
+        qudt:exponent -1
+        ];
+    rdfs:label "Swedish Krona per Euro" ;
+    qudt:symbol "SEK/EUR" ;
+    qudt:plainTextDescription "Exchange rate of Swedish Krona to Euros" ;
+    rdfs:isDefinedBy <http://qudt.org/vocab/unit> ;
+.
+
+quantitykind:ExchangeRate
+    a qudt:QuantityKind ;
+    qudt:applicableUnit unit:CCY_NOK-PER-CCY_EUR ;
+    qudt:applicableUnit unit:CCY_SEK-PER-CCY_EUR ;
+    qudt:applicableUnit unit:CCY_DKK-PER-CCY_EUR ;
+    qudt:hasDimensionVector qkdv:A0E0L0I0M0H0T0D1 ; ## or D1 ?
```

Comment:
replace the comment `or D1 ?` (it's silly because the object is in fact D1) with a link to github issue